### PR TITLE
feat: add support to default activity

### PR DESCRIPTION
**Pull request by uziasferreirazup**
### Description and Example

Remove the necessity to create an activity to use beagle

### Checklist

Please, check if these important points are met using `[x]`:

- [X] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [X] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [X] I am willing to follow-up on review comments in a timely manner.

[PR Guide]: https://github.com/ZupIT/beagle/blob/master/doc/contributing/pull_requests.md

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/beagle/src/main/res/layout/beagle_include_default_error_server_driven.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:padding="16dp">
+
+    <ImageView
```
Comment (botbeaglezup) [inline android/beagle/src/main/res/layout/beagle_include_default_error_server_driven.xml:8]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle/blob/82bcc147d51a4912b09d41e7c3387c025df5890c/android/beagle/src/main/res/layout/beagle_include_default_error_server_driven.xml#L8"></span>Missing <code>contentDescription</code> attribute on image</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/82bcc147d51a4912b09d41e7c3387c025df5890c/android/sample/src/main/AndroidManifest.xml#L22">android/sample/src/main/AndroidManifest.xml#L22</a> - On SDK version 23 and up, your app data will be automatically backed up and restored on app install. Consider adding the attribute <code>android:fullBackupContent</code> to specify an <code>@xml</code> resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/82bcc147d51a4912b09d41e7c3387c025df5890c/android/sample/src/main/AndroidManifest.xml#L30">android/sample/src/main/AndroidManifest.xml#L30</a> - Attribute <code>usesCleartextTraffic</code> is only used in API level 23 and higher (current min is 19)</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by matheusribeirozup** [comment]
Please remember to add the `documentation` tag and update everywhere you refer to `ServerDrivenActivity`.

**Review by matheusribeirozup — APPROVED** [review]

**Review by hernandazevedozup — APPROVED** [review]

**Pull request merged**